tests/models/bark/test_modeling_bark.py

```diff
@@ -72,8 +74,6 @@ def __init__(
         initializer_range=0.02,
         n_codes_total=8,  # for BarkFineModel
         n_codes_given=1,  # for BarkFineModel
-        config_class=None,
-        model_class=None,
```

Comment:
There is no need to have these arguments. We can just set it explicitly in each model tester classes.